OrbitGl/SamplingReportDataView.cpp

```diff
@@ -259,13 +265,13 @@ void SamplingReportDataView::DoFilter() {
   for (size_t i = 0; i < functions_.size(); ++i) {
     SampledFunction& func = functions_[i];
     std::string name = ToLower(func.name);
-    std::string module = ToLower(func.module);
+    std::string module_name = ToLower(Path::GetFileName(func.module_path));
```

Comment:
is this intended? Maybe users want to search in path?

Reply:
Well at least its the current behaviour.

Reply:
Yes, it is current behaviour, but more importantly, we only display the name and not the full path. It makes sense to me to only filter things that are displayed